subsys/bluetooth/controller/ll_sw/ull_df.c

```diff
@@ -193,6 +193,18 @@ uint8_t ll_df_set_cl_cte_tx_params(uint8_t adv_handle, uint8_t cte_len,
 	return BT_HCI_ERR_SUCCESS;
 }
 
+/* @brief Function enables of disables CTE TX for periodic advertising.
```

Comment:
Enables or disables.

Reply:
I'll leave it as is. Every other @brief in this file start with "Function", so I want to be consistent.

Reply:
I was correcting the spelling: of -> or.

Reply:
Sorry for misunderstanding, I've corrected it.